components/automate-cli/cmd/chef-automate/upgrade.go

```diff
@@ -267,6 +289,26 @@ func statusUpgradeCmd(cmd *cobra.Command, args []string) error {
 				writer.Printf("Automate is up-to-date (%s)\n", resp.LatestAvailableVersion)
 			}
 		}
+
+		_, isMajorVersion := manifest.IsSemVersionFmt(resp.CurrentVersion)
+		if isMajorVersion {
+			ci, err := majorupgradechecklist.NewPostChecklistManager(resp.CurrentVersion)
+			if err != nil {
+				return err
+			}
+			resp, err := ci.ReadPendingPostChecklistFile()
+			if err != nil {
+				return status.Wrap(
+					err,
+					status.DeploymentServiceCallError,
+					"unable to read checklist file",
+				)
+			}
+			for index, msg := range resp {
+				writer.Body("\n" + strconv.Itoa(index+1) + ") " + msg)
+			}
+		}
```

Comment:
because we created about function GetPendingPostChecklist, we can now leverage it here.
```go
pendingPostChecklist, err := GetPendingPostChecklist()
if err != nil {
        return status.Wrap(err, status.DeploymentServiceCallError, "unable to read checklist file")
}
for index, msg := range pendingPostChecklist {
        writer.Body("\n" + strconv.Itoa(index+1) + ") " + msg)
}
```